neo/VM/JsonParser.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Neo.IO.Json;
+using System;
+using System.Numerics;
+using System.Text;
+
+namespace Neo.VM
+{
+    public static class JsonParser
+    {
+        /// <summary>
+        /// Convert stack item in json
+        /// </summary>
+        /// <param name="item">Item</param>
+        /// <returns>Json</returns>
+        public static JObject Serialize(StackItem item)
+        {
+            switch (item)
+            {
+                case VM.Types.Array array:
+                    {
+                        var ret = new JArray();
+
+                        foreach (var entry in array)
+                        {
+                            ret.Add(Serialize(entry));
+                        }
+
+                        return ret;
+                    }
+                case VM.Types.ByteArray buffer:
+                    {
+                        return new JString(buffer.GetString());
+                    }
+                case VM.Types.Integer num:
+                    {
+                        var integer = num.GetBigInteger();
+                        if (integer > ulong.MaxValue) return new JString(integer.ToString());
```

Comment:
The max value of `number` in JavaScript is <code>2<sup>53</sup> - 1</code>.

Reply:
`double.MaxValue`?

Reply:
<code>2<sup>53</sup> - 1</code>